### DISCUSSION TITLE
Question: How to update game for new week?

### DISCUSSION THREAD

**Discussion by shecravesaldo** [FunkinCrew/Funkin#1483] [Q&A] [answered]
I was just wondering, when the game is updated on the site (newgrounds or itch), do I have to redownload the game again to get the new update or will it automatically send to my computer?

**Comment by ghost** [comment]
> I was just wondering, when the game is updated on the site (newgrounds or itch), do I have to redownload the game again to get the new update or will it automatically send to my computer?

I usually redownload the game as this is a more obvious and easy approach. not sure If you can update the game automatically.

**Comment by SilverT4** [comment]
yes, you have to redownload the game to get the new update.

**Comment by superguyoffa** [comment, marked as answer]
> > I was just wondering, when the game is updated on the site (newgrounds or itch), do I have to redownload the game again to get the new update or will it automatically send to my computer?
> 
> I usually redownload the game as this is a more obvious and easy approach. not sure If you can update the game automatically.

You can use the itch.io launcher, it is basically like steam but for itch.io, which means it will also automatically update Friday Night Funkin'

Itch.Io Launcher: https://itch.io/app (or search up Itch.io launcher on your browser)
